draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -0,0 +1,1568 @@
+          base physical-isolation;
+          description
+            "Process and threads isolation.";
+        }
+        identity physical-network-isolation {
+          base physical-isolation;
+          description
+            "Process and threads isolation.";
+        }
+        identity virtual-resource-isolation {
+          base logical-isolation;
+          description
+            "A network slice has access to specific range of resources
+             that do not overlap with other network slices
+             (e.g. VM isolation).";
+        }
+        identity network-functions-isolation {
+          base logical-isolation;
+          description
+            "NF (Network Function) is dedicated to the network slice,
+             but virtual resources are shared.";
+        }
+        identity service-isolation {
+          base logical-isolation;
+          description
+            "NSC data are isolated from other NSCs, but virtual
+             resources and NFs are shared.";
+        }
+
+
+
+Liu, et al.              Expires 13 August 2023                [Page 13]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+        identity resize-option {
+          description
+            "Base identity for link or connectivity resizing options";
+        }
+
+        identity resize-none {
+          base resize-option;
+          description
+            "Not resizable";
+        }
+
+        identity resize-with-hit {
+          base resize-option;
+          description
+            "Resizable with traffic hits";
+        }
+
+        identity resize-hitless {
+          base resize-option;
+          description
+            "Hitless resizable";
+        }
+
+        /*
+         * Groupings
+         */
+
+        grouping ns-topo-slo-sle-policy {
+          description
+            "Policy grouping for Transport Network Slices.";
+
+          leaf optimization-criterion {
+              type identityref {
+                base te-types:objective-function-type;
+              }
+              description
+                "Optimization criterion applied to this topology.";
+          }
+          leaf delay-tolerance {
+            type boolean;
+            description
+              "'true' if is not too critical how long it takes to
+               deliver the amount of data.";
+            reference
+              "GSMA-NS-Template: Generic Network Slice Template,
+               Version 3.0.";
+          }
+          leaf-list periodicity {
+
+
+
+Liu, et al.              Expires 13 August 2023                [Page 14]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+            type uint64;
+            units seconds;
+            description
+              "A list of periodicities supported by the network
+               slice.";
+            reference
+              "GSMA-NS-Template: Generic Network Slice Template,
+               Version 3.0.";
```

Comment:
For instance, this kind of parameters probably makes no sense now, to be checked